web/app/scripts/src/directives/form.es.js

```diff
@@ -30,6 +30,17 @@ module.directive('surveysForm', [
             templateUrl: 'templates/incuna-surveys/form/survey-form.html',
             link: function (scope) {
                 scope.model = {};
+                scope.percentageComplete = 0 + '%';
+
+                const countQuestionsTotal = function () {
+                    const questions = scope.form.fieldsets;
+                    scope.totalQuestionCount = 0;
+
+                    angular.forEach(questions, function (question) {
```

Comment:
We seem to be using to now everywhere, so for consistency. Other than I think it just reads cleaner:
`questions.forEach` instead of `angular.forEach(question,...)`